README.md

```diff
@@ -114,6 +130,6 @@ Your Answer:
 Select all that apply:
 ```
 [] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
-[] Synchronous code runs all at the same time, asynchronous code runs completely randomly
+[x] Synchronous code runs all at the same time, asynchronous code runs completely randomly
 [] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
```

Comment:
the answer here is the last one, `Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.` I like to think of synchronous code as executing line-by-line, and therefore is blocking as it will not run the next line until the last line has been completed. In comparison,  asych code is non-blocking, so the rest of the program will run without waiting for the block of code to execute.